I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 530, in script
    teach "Everybody sit down!"
IOError: Couldn't find file 'bell.wav'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 530, in script
    teach "Everybody sit down!"
  File "renpy/ast.py", line 901, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "renpy/exports.py", line 1373, in say
    who(what, *args, **kwargs)
  File "renpy/character.py", line 1253, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "renpy/character.py", line 917, in do_display
    **display_args)
  File "renpy/character.py", line 653, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3376, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "renpy/display/core.py", line 4096, in interact_core
    renpy.audio.audio.periodic()
  File "renpy/audio/audio.py", line 1166, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "renpy/audio/audio.py", line 1144, in periodic_thread_main
    periodic_pass()
  File "renpy/audio/audio.py", line 1086, in periodic_pass
    c.periodic()
  File "renpy/audio/audio.py", line 517, in periodic
    topf = load(filename)
  File "renpy/audio/audio.py", line 70, in load
    rv = renpy.loader.load(fn)
  File "renpy/loader.py", line 820, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'bell.wav'.

Windows-10-10.0.19041 AMD64
Ren'Py 7.5.0.22062402
Bystander 1.0
Fri Jul 15 03:55:42 2022
